spec/integration/user_can_make_bid_when_current_high_bid_exists_spec.rb

```diff
@@ -0,0 +1,58 @@
+require 'rails_helper'
+
+RSpec.feature "user can place a bid on an item when a bid already exists" do
+  scenario "they create a second bid for an item" do
+    item = create(:item)
+    user1 = create(:user)
+    user2 = create(:user)
+
+    allow_any_instance_of(ApplicationController).to receive(:current_user).and_return(user1)
+
+    visit item_path(item)
+    expect(page).to have_content("No current bids")
+    fill_in "bid[price]", with: "10.99"
+    click_button("Place Bid")
```

Comment:
if we have time, we could refactor this b/c technically we don't need to have a user go through the submitting a bid process, since we've already tested this functionality. we could stub out the process so we can make sure the test is just testing whether a user can place a bid when one already exists